src/test/java/ca/corefacility/bioinformatics/irida/ria/integration/projects/ProjectShareSamplesIT.java

```diff
@@ -119,4 +117,35 @@ public void testShareSamplesAsManager() {
 		assertEquals(1, shareSamplesPage.numberOfSamplesWithSameNames(),
 				"There should be one sample listed which exists in the target project with the same name and different identifier");
 	}
+
+	@Test
+	void testSharingWithALockedSample() {
+		final String LOCKED_SAMPLE_NAME = "sample5fdgr";
+
+		LoginPage.loginAsManager(driver());
+
+		// SHARING SINGLE SAMPLE
+		ProjectSamplesPage samplesPage = ProjectSamplesPage.goToPage(driver(), 1);
+		samplesPage.selectSampleByName("sample5fg44");
+		samplesPage.selectSampleByName(LOCKED_SAMPLE_NAME);
+		samplesPage.shareSamples();
+
+		assertFalse(shareSamplesPage.isNextButtonEnabled(), "");
```

Comment:
Can you add a message here instead of just an empty string

Reply:
Updated in fabffa8

Also added the missing i18n string in ab8ca1e